companies/apple.json

```diff
@@ -9,10 +9,12 @@
     "name": "Apple Distribution International",
     "address": "Hollyhill Industrial Estate\nHollyhill, Cork\nIreland",
     "email": "contactus.de@euro.apple.com",
+    "phone": "+353 21 428 4000",
     "web": "https://www.apple.com",
     "sources": [
         "https://www.apple.com/legal/privacy/de-ww/",
-        "https://www.apple.com/de/contact/"
+        "https://www.apple.com/de/contact/",
+        "https://britainfirm.com/ireland/apple-distribution-international-542875"
```

Comment:
This unfortunately seems like quite a spammy and unreliable site. We would need one preferably from the official Apple site or, barring that, from a trusted third-party site.

Reply:
Thanks for your concern, now I added the verified contacts from the official site of Apple with the proper source.